### PR TITLE
Add documentation to TimeseriesSearch

### PR DESCRIPTION
**Pull request by follan**

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-152.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/stories/full.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+| Property                     | Description | Type     |
+| ---------------------------- | ----------- | -------- |
+| `onTimeserieSelectionChange` | Callback function called when the selection changes. Called with two parameters: the current list of selected ids and the last added/removed timeseries   | `(newTimeseriesIds: number[], selectedTimeseries: Timeseries) => void` |
+
+##### Optional:
+
+| Property             | Description                                                                  | Type                                      | Default     |
+| -------------------- | ---------------------------------------------------------------------------- | ------------------------------------------| ----------- |
+| `selectedTimeseries` | List of preselected timeseries                                               | `number[]`                                | `[]`        |
+| `rootAsset`          | The selected root asset id. `undefined` will select all                      | `number`                                  | `undefined` |
```
Comment (mrtraser) [inline src/components/TimeseriesSearch/stories/full.md]: Guess we can leave cells blank in case of undefined default value

Comment (follan) [reply src/components/TimeseriesSearch/stories/full.md]: Fixed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-152.gearboxjs.preview.cogniteapp.com

**Pull request merged**